ui/src/main/resources/org/pentaho/di/ui/spoon/job/messages/messages_en_US.properties

```diff
@@ -247,3 +247,9 @@ JobGraph.Dialog.LoopAfterHopEnabled.Message=Enabling this hop creates a loop tha
 JobGraph.Dialog.HopFlipCausesLoop.Message=Flipping the direction of this hop creates a loop that might cause errors when this job runs.
 JobGraph.Dialog.HopCausesLoop.Message=Adding this hop creates a loop that might cause errors when this job runs.
 JobGraph.Dialog.HopCausesLoop.Title=Warning
+
+JobGraph.DeprecatedEntry.Tooltip.Title=This is a deprecated step and will be removed.
+JobGraph.DeprecatedEntry.Tooltip.Message1={0} will be removed in an upcoming
```

Comment:
Out of curiosity, is it not possible to provide this text as a single message with new line characters in appropriate places?